### DISCUSSION TITLE
how to limit logging of connecting users

### DISCUSSION THREAD

**Discussion by mothabrain** [jellyfin/jellyfin#11682] [General] [answered]
is there a way to limit the logging of incoming connections? where can i find the specific db files? it would be a nice feature to limit the logging time based, like older than 1 week can be deleted.

Thanks for tips in advance

**Comment by crobibero** [comment]
What logging are you referring to?

**Reply by mothabrain** [reply]
mainly the protocol of users who logged in with time and ip address

**Reply by crobibero** [reply]
In `system.xml` you can edit `ActivityLogRetentionDays`, and enable the scheduled task to automatically remove activity log entries

**Reply by mothabrain** [reply]
much thanks for your quick reply. 

i will try it out and check if it works how i expect it

**Reply by mothabrain** [reply]
just check the xml:

`  <ActivityLogRetentionDays>30</ActivityLogRetentionDays>`

but the db goes back further than 30 days... :/

**Reply by crobibero** [reply, marked as answer]
Did you enable the scheduled task?

**Comment by mothabrain** [comment]
that did the job! thanks!
